src/hunit/text/indent.go

```diff
@@ -20,15 +25,20 @@ func Indent(s, p string) string {
  * Indent a string by prefixing each line with the provided string
  */
 func IndentWithOptions(s, p string, opt IndentOptions) string {
-  var o string
+	var o bytes.Buffer
   if (opt & IndentOptionIndentFirstLine) == IndentOptionIndentFirstLine {
-    o += p
+    o.WriteString(p)
   }
   for i := 0; i < len(s); i++ {
-    o += string(s[i])
+		o.WriteString(string(s[i]))
     if s[i] == '\n' {
-      o += p
+      o.WriteString(p)
     }
   }
-  return o
+  return o.String()
+}
+
+func IsJSON(s string) bool {
```

Comment:
Try to change this to something like:

```go
func HasContentType(req *http.Request, t string) bool {
  contentType := req.Header.Get("Content-Type")
  return // figure out if it is...
}
```

It probably belongs in the file `http.go` and not `indent.go`, as well.